hackathon/views.py

```diff
@@ -550,3 +591,87 @@ def assign_mentors(request, hackathon_id):
             messages.error(request,
                            (f"An unexpected error occurred: "
                             f"{hack_mentors_formset.errors}"))
+
+
+def event_list(request, hackathon_id):
```

Comment:
This is not secure. You are missing the `@login_required` and `@can_access` view decorators here.